torch/csrc/jit/codegen/cuda/kernel_ir.cpp

```diff
@@ -22,7 +22,7 @@ void Node::print() const {
 }
 
 Val::Val(Passkey passkey, DataType dtype) : Node(passkey), dtype_(dtype) {
-  id_ = passkey.kernel->newValueId(passkey);
+  id_ = passkey.kernel->newValueId(passkey); // NOLINT
```

Comment:
we can add a link to the clang-tidy bug to explain the need for suppressing the diagnostic:
https://bugs.llvm.org/show_bug.cgi?id=48534

Reply:
Changed to use `NOLINTNEXTLINE` as adding the link triggers a new line by clang-format.